debian/changelog

```diff
@@ -1,7 +1,70 @@
-ubuntu-advantage-tools (27.2.2~21.10) impish; urgency=medium
+ubuntu-advantage-tools (27.3~21.10.1) impish; urgency=medium
 
   * d/tools.postinst:
-    - Do not fail in postinst if cloud-init did not run. (LP: #1936833)
+    - handle any cloud-id error and treat as "none" cloud
+    - purge old ua-messaging.timer/service
+  * systemd:
+    - add new ua-timer.timer that runs every 2 hours
+    - add new ua-license_check.timer that runs every 5 minutes only if
+      activated by ua-license-check.path
+  * New upstream release 27.3
+    - ros:
+      + add beta support to enable ros and ros-updates
+      + add support for "required services" so that esm-infra and esm-apps
+        get auto-enabled when enabling ros or ros-updates
+      + add support for "dependent services" so that user gets prompted to
+        disable ros/ros-updates if they disable esm-infra/esm-apps
+    - fips:
+      + allow fips on GCO bionic now that optimized kernel is ready
+      + disallow enabling fips on focal on clouds until cloud-optimized focal
+        fips-certified kernel is ready (LP: #1939449, LP: #1939932)
+      + print warning if cloud-id fails about installing generic fips kernel
+    - cloud:
+      + rely only on cloud-id to determine cloud type (LP: #1940131)
+      + catch errors when determining cloud type (LP: #1938207) (GH: #1541)
+    - azure:
+      + bump IMDS API version to support Azure published images
+    - proxy:
+      + shows better error message when a port value in a proxy is invalid
+    - cli:
```

Comment:
(maybe cli is not exactly the place for this but) I think we should include in the changelog that we fixed the caches, making some of them permanent - I think this bug has (low, but still has) relevance to the users